frontend/src/metabase/query_builder/components/view/ViewHeader.styled.jsx

```diff
@@ -53,3 +54,38 @@ export const DatasetHeaderButtonContainer = styled.div`
   position: relative;
   right: 0.3rem;
 `;
+
+export const HeaderButton = styled(Button)`
+  background-color: ${({ active, color = getDefaultColor() }) =>
+    active ? alpha(color, 0.8) : "transparent"};
+  color: ${({ active }) => (active ? "white" : color("text-dark"))};
+  &:hover {
+    background-color: ${({ active, color = getDefaultColor() }) =>
+      active ? alpha(color, 0.8) : "transparent"};
+    color: ${({ active }) => (active ? "white" : color("text-dark"))};
+  }
+  transition: background 300ms linear, border 300ms linear;
```

Comment:
While this is likely subtle enough, doesn't hurt to support reduced motion (see #18169).

Reply:
Nice catch! Fixed